app/controllers/api/v1/sessions_controller.rb

```diff
@@ -5,13 +5,12 @@ class Api::V1::SessionsController < Api::V1::BaseController
   skip_before_action :authenticate_user_using_x_auth_token
 
   def create
-    user = User.find_for_database_authentication(email: params[:user] && params[:user][:email])
+    user = User.find_for_database_authentication(email: session_params[:email])
     if invalid_password?(user)
-      respond_with_error "Incorrect email or password", 401
+      respond_with_error(t("invalid_credentials"), :unauthorized)
     else
       sign_in(user)
-      render json: { auth_token: user.authentication_token, user: user, is_admin: user.super_admin? },
-        location: root_path, status: :created
+      respond_with_json(:created, { auth_token: user.authentication_token, user: user, is_admin: user.super_admin? })
```

Comment:
I would prefer to keep it aligned with granite. https://github.com/bigbinary/granite/blob/master/app/controllers/concerns/api_responders.rb#L18-L20